AsyncDisplayKit/ASTableView.mm

```diff
@@ -1454,6 +1463,33 @@ - (void)rangeController:(ASRangeController *)rangeController didEndUpdatesAnimat
   }
 }
 
+- (void)rangeControllerDidReloadData:(ASRangeController *)rangeController
+{
+  ASDisplayNodeAssertMainThread();
+  LOG(@"UITableView reloadData");
+  
+  if (!self.asyncDataSource) {
+    return; // if the asyncDataSource has become invalid while we are processing, ignore this request to avoid crashes
+  }
+  
+  ASPerformBlockWithoutAnimation(YES, ^{
+    if (self.test_enableSuperUpdateCallLogging) {
+      NSLog(@"-[super reloadData]");
+    }
+    [super reloadData];
+    if (!_performingBatchUpdates) {
+      [_rangeController updateIfNeeded];
+    }
+    // TODO Should reloadData triggers batch fetching?
+//    [self _scheduleCheckForBatchFetchingForNumberOfChanges:];
```

Comment:
I believe it should. If I reload data, and I don't have e.g. 2 screenfuls of data ahead of me, I should attempt to batch fetch.